lib/cmds/space_cmds/create.ts

```diff
@@ -43,11 +44,6 @@ module.exports.builder = yargs => {
       describe:
         'Confirm space creation without prompt, be aware this may result in extra monthly charges depend on your subscription'
     })
-    .option('default-locale', {
```

Comment:
@massao this seems to be removed by mistake?

Reply:
Actually this is because the `createSpace` function doesn't accept the `defaultLocale` param.
As you can see here: https://github.com/contentful/contentful-management.js/blob/ea3b20fd431544b9a35fc40ee23c3f7c747d3383/lib/create-contentful-api.ts#L102-L105
It expects `spaceData: Omit<SpaceProps, 'sys'>`
And from here you can see that `SpaceProps` having the `sys` omitted is just `name`: https://github.com/contentful/contentful-management.js/blob/ea3b20fd431544b9a35fc40ee23c3f7c747d3383/lib/entities/space.ts#L8-L11

Reply:
huh, good catch. so weird it was never working! cc @marcolink

Reply:
True, it was being [passed](https://github.com/contentful/contentful-management.js/blob/master/lib/create-contentful-api.ts#L110) nonetheless. We will to put it back in another PR, thanks.

Here's a PR to fix the types https://github.com/contentful/contentful-management.js/pull/1474